linux_os/guide/system/software/integrity/certified-vendor/installed_OS_is_FIPS_certified/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: fedora,ocp4,ol7,ol8,rhel6,rhel7,rhel8,rhv4
+prodtype: fedora,ocp4,ol7,ol8,rhel6,rhel7,rhel8,rhv4,sle12,sle15
```

Comment:
To make specific mention: SLE15 is on the NIST Modules in Process list (https://csrc.nist.gov/Projects/cryptographic-module-validation-program/Modules-In-Process/Modules-In-Process-List) and Implementation Under Test list (https://csrc.nist.gov/Projects/cryptographic-module-validation-program/Modules-In-Process/IUT-List)

For that reason, adding SLE15 seems appropriate. Was the same gate RHEL8 was held against.

Reply:
I have it in the list, but removed it from the check phase as it is still in process.  I at least want it to return a negative until it is certified

Reply:
Totally your call. RHEL8 was only added after being formally listed as in progress by NIST. 

We should hold other distros to the same standard/gate. If you feel its better to hold off, no worries. I will also merge if you want to add it now.

Reply:
I'll add it and push.  That way we don't have to waste review cycles in the future on such a small changes